apps/vessel-history/features/profile/components/Info.tsx

```diff
@@ -78,9 +77,10 @@ const Info: React.FC<InfoProps> = (props): React.ReactElement => {
       value: Math.floor(i.length('days')),
     })
     setLoading(true)
-    await dispatchDeleteOfflineVessel(data)
+    dispatchDeleteOfflineVessel(data)
+    dispatchFetchOfflineVessels()
```

Comment:
The `dispatchDeleteOfflineVessel` should already update the redux store, why calling `dispatchFetchOfflineVessels` right after?